doc/guides/getting_started/src/README.md

```diff
@@ -59,7 +59,7 @@ A number of software packages from the distribution's package manager are requir
 On Ubuntu 20.04, the required packages can be installed with the following command.
 
 ```sh
-sed -e '/^$/d' -e '/^#/d' ./apt-requirements.txt | sudo apt install -y
+sed -e '/^$/d' -e '/^#/d' ./apt-requirements.txt | xargs sudo apt install -y
```

Comment:
Nice!

Optional: while you're in there, I think we can simplify the sed command. Xargs will ignore blank lines, so we can delete `-e '/^$/d'`. And now there's only one command, so no need for `-e`.

Et voila:

```sh
sed '/^#/d' apt-requirements.txt | xargs sudo apt install -y
```